content/en/examples/application/web/web.yaml

```diff
@@ -29,7 +29,7 @@ spec:
     spec:
       containers:
       - name: nginx
-        image: registry.k8s.io/nginx-slim:0.7
+        image: registry.k8s.io/nginx-slim:0.24
```

Comment:
I'm not sure this works. We need two **different** image tags; they can be any two tags, but they can't be the same as each other.